res/TensorFlowLiteRecipes/Part_Sqrt_Rsqrt_002_Selected_01/test.recipe

```diff
@@ -0,0 +1,37 @@
+operand {
```

Comment:
`Part_xxx` recipes are for circle-partitioner so please use other prefix for opselector like `OpSel_xxx`.